setup.cfg

```diff
@@ -0,0 +1,39 @@
+[flake8]
+# Copied from https://github.com/XENONnT/straxen/blob/master/setup.cfg
+# Set maximum width of the line to 100
+max-line-length = 100
+
+# Excluding some directories:
+exclude =
+  .git
+  .github
+  docs*
+  notebooks*
+  *.yml
+  __pycache__
+  .venv
+  .eggs
+  *.egg
+  dist
+  *cfg
+
+
+# E203 whitespace before ':'
+# F401 imported but unused
+# F403 unable to detect undefined names
+# W503 line break before binary operator
+
+ignore = E203, W503
+
+per-file-ignores =
+    GOFevaluation/*__init__.py: F401, F403
+
+[docformatter]
+in-place = true
+blank = true
+style = sphinx
```

Comment:
alea used google style: https://github.com/XENONnT/alea/blob/13043e57b70c0fbb849f8a2ac5a0439d1714b228/setup.cfg#L34. Is this decision to use sphinx mainly for not changing too many lines of code, or do you have a preference on sphinx?

Reply:
We made the decision for sphinx style a while ago, I think since straxen uses it. I don't like it overly but I think there is no need to change this now really.

Reply:
Yes, no problem. Reasonable decision.